packages/common/src/groups/getWellKnownGroup.ts

```diff
@@ -0,0 +1,19 @@
+import { WellKnownGroups } from "./_internal/WellKnownGroups";
+import { cloneObject } from "../util";
+import { IHubGroup } from "../core/types/IHubGroup";
+
+/**
+ * Fetches a well known group template based on a name
+ *
+ * TODO: This should be expanded to check for privs and further flesh out
```

Comment:
minor, but do we still need this TODO? Or was it decided we no longer needed that sort of complexity?